fluentui_drawer/src/main/java/com/microsoft/fluentui/drawer/DrawerDialog.kt

```diff
@@ -59,6 +57,8 @@ open class DrawerDialog @JvmOverloads constructor(context: Context, val behavior
 
     private var isExpanded: Boolean = false
     protected val container: View
+    protected val drawerContent: ViewGroup
+    private val drawer: View
```

Comment:
Assuming these changes are unrelated to AndroidX update, right?

Reply:
These are actually, as we cannot reference views directly now and it is deprecated. For reference https://developer.android.com/topic/libraries/view-binding/migration